packages/eslint-plugin-pf-codemods/src/rules/v6/chipReplaceWithLabel/chip-replace-with-label.ts

```diff
@@ -0,0 +1,202 @@
+import { getFromPackage } from '../../helpers';
+import {
+  ImportDeclaration,
+  ImportSpecifier,
+  JSXAttribute,
+  JSXClosingElement,
+  JSXElement,
+  JSXExpressionContainer,
+  JSXFragment,
+  Literal,
+  Node,
+} from 'estree-jsx';
+import { Rule } from 'eslint';
+
+type JSXAttributeValue =
+  | Literal
+  | JSXExpressionContainer
+  | JSXElement
+  | JSXFragment;
+
+const getImportSpecifier = (node: ImportDeclaration, specifierName: string) => {
+  return node.specifiers.find(
+    (specifier) =>
+      specifier.type === 'ImportSpecifier' &&
+      specifier.imported.name === specifierName
+  ) as ImportSpecifier | undefined;
+};
+
+const renameImportSpecifier = (
+  importToRename: ImportSpecifier | undefined,
+  node: ImportDeclaration,
+  fixer: Rule.RuleFixer,
+  newName: string
+) => {
+  if (!importToRename) {
+    return [];
+  }
+
+  const oldName = importToRename.imported.name;
+  const importSpecifier = getImportSpecifier(
+    node,
+    importToRename.imported.name
+  ) as ImportSpecifier;
+
+  const importedName = importSpecifier.imported.name;
+  const name = importedName.replace(oldName, newName);
+  const localName = importSpecifier.local.name;
+  const aliasText = importedName !== localName ? ` as ${localName}` : '';
+  const rename = `${name}${aliasText}`;
```

Comment:
Rename should be used instead of `newName` in the fixer, thanks for the catch.

Reply:
I think you can keep the `newName` there as it can make things easier afterwards when renaming the JSXElement, so you don't have to check if an alias was used there. But it is up to you. Also the duplicate Label import might play a role, so it might end up more complicated.